layouts/index.html

```diff
@@ -20,7 +20,7 @@ <h5>{{ .Site.Params.subHeading }}</h5>
 
             <div class="header_right">
                 <div class="rocket">
-                    <img src="./images/pymc-logo-350.png" alt="">
+                    <img src="./images/pymc-logo-350.png" alt="PyMC Rocket Logo in motion upward">
```

Comment:
I guess we should take note of @OriolAbril's comment [here](https://github.com/pymc-devs/pymcon_2022/pull/70#discussion_r950729302).